### PR TITLE
Some core linting

### PR DESCRIPTION
**Pull request by felixrindt**

### CODE REVIEW

File: packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/recipientsFrom..st [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-	aJsonObject ifNil: [^ nil].
-	
-	recipients := OrderedCollection new.
-	aJsonObject do: [:eachRecipient | recipients add: eachRecipient].
-	
-	^ recipients
\ No newline at end of file
+	aJsonObject
+		ifNil: [^ nil]
+		ifNotNil: [^ aJsonObject asOrderedCollection]
```
Comment (felixrindt) [inline packages/IMAPClient-Core.package/ICFileReadWriter.class/instance/recipientsFrom..st:6]: Der ist besonders schön :D

File: packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st [hunk 2]
```diff
@@ -1,4 +1,10 @@
 printing
 printOn: aStream
 
-	aStream nextPutAll: self accountName, ' ', self host, ' ', self port, ' ', self username, ' ', self password, ' ', self ssl
\ No newline at end of file
+	aStream nextPutAll:
```
Comment (timgarrels) [inline packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st]: Use `joinSeparatedBy:` here instead of string, blank, string, blank

**Review by timgarrels — CHANGES_REQUESTED** [review]

File: packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st (hunk 2 above)
Comment (felixrindt) [reply packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st]: image
Looks like this now... I think this is only slightly better :/
Any better ideas @EatingBacon ?

Comment (timgarrels) [reply packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st]: A newline after every dot?
``` smalltalk
aStream nextPut: ({
    self username.
    ...
    self password} joinSeparatedBy: '')
```

Comment (felixrindt) [reply packages/IMAPClient-Core.package/ICAccountInfo.class/instance/printOn..st]: fair :)

**Review by timgarrels — APPROVED** [review]

**Pull request merged**